### PR TITLE
Updated Alert Colour for Pilot Msg

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#13390

*Description of changes:*
* Update alert text colour as per UX for accessibility

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DigitalCredentials/IssueCredentials.vue [hunk 1]
```diff
@@ -121,7 +121,7 @@ export default class IssueCredentials extends Vue {
     }
   }
   .alert-text {
-    color: $BCgovGold9;
+    color: #F8661A;
```
Comment (severinbeauvais) [inline src/components/DigitalCredentials/IssueCredentials.vue:124]: Should use named colour (CSS variable).

Also please update app version.

Comment (severinbeauvais) [reply src/components/DigitalCredentials/IssueCredentials.vue:124]: Make sense but please add it anyway.

Is it different from existing red?

**Review by severinbeauvais — APPROVED** [review]
Great variable name 👍

**Comment by severinbeauvais** [comment]
@argush3 FYI - will be merged soon, and can go in latest release I think

File: src/components/DigitalCredentials/IssueCredentials.vue (hunk 1 above)
Comment (cameron-freshworks) [reply src/components/DigitalCredentials/IssueCredentials.vue]: It is unique, yes 😬 
It's a very special message!

**Pull request merged**